src/xcode/ENA/ENA/Source/Models/Task Scheduling/ENATaskScheduler.swift

```diff
@@ -62,12 +66,18 @@ final class ENATaskScheduler {
 
 	func scheduleTasks() {
 		scheduleTask(for: .primaryBackgroundTask, cancelExisting: true)
+		scheduleTask(for: .secondaryBackgroundTask, cancelExisting: true)
 	}
 
 	func cancelTasks() {
 		BGTaskScheduler.shared.cancelAllTaskRequests()
```

Comment:
If you cancel all task requests, doesn't it mean that you would be canceling the `secondaryBackgroundTask` from the `primaryBackgroundTask` and vice versa? Do you really want this?

Reply:
👍